src/rules/declaration-block-no-ignored-properties/README.md

```diff
@@ -0,0 +1,53 @@
+# declaration-block-no-ignored-properties
+
+Disallow properties has no effect.
```

Comment:
Same suggestion as above: "Disallow properties that are ignored due to another property value in the same rule."